### ISSUE TITLE
Toot's menu is unrolling behind images, making it unusable

### ISSUE THREAD

**Issue by ZeldaMoMa** [glitch-soc/mastodon#1420]
In the web UI, I expect toot's menu to unroll in front of everything else to be usable.
But when I unroll a menu on a toot containing one or several images or videos (It seems ok with audio) and if the menu unroll upward, it is hidden by the images, the videos or the "sensitive content" thing.
The bug is present with the glitch-soc "letterbox media" setting checked and unchecked.

My instance version is v3.2.0+glitch, I'm using Firefox 68.5

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
  - I have search with the keyword "menu" only, I have checked mastodon's issues as well.
- [ ] This bugs also occur on vanilla Mastodon
  - I would think so, but I can't be sure.

![bugmastoimage1bis](https://user-images.githubusercontent.com/48503235/92739797-8a998580-f37d-11ea-8e3f-04f2667a8103.png)
![bugmastoimage2bis](https://user-images.githubusercontent.com/48503235/92739805-8c634900-f37d-11ea-8072-467f137576c3.png)
![bugmastoimage3bis](https://user-images.githubusercontent.com/48503235/92739815-8f5e3980-f37d-11ea-8a4b-09812da6317d.png)
![bugmastoimage4bis](https://user-images.githubusercontent.com/48503235/92739822-908f6680-f37d-11ea-93c0-5f65c4287e08.png)
![bugmastoimage5bis](https://user-images.githubusercontent.com/48503235/92739835-938a5700-f37d-11ea-90da-c672d8c4b440.png)

**Comment by ThibG** [comment]
This is surprising and something I can't reproduce.
Do you have any custom CSS? Any extension that could mess with this?
Also, can you use Firefox's “Inspect Element” on images and see if there is something in their CSS with some `z-index` property set?

**Comment by ZeldaMoMa** [comment]
No custom CSS that I know of and I can reproduce the bug on and other glitch-soc instance.
I've disabled all my add-ons and still have the problem.

There is no `z-index` properties on the images, the only thing with a `z-index: -1000 !important;` property, it's just before the menu part in the html.
Also, when the menu is unrolled, I cant, use "Inspect Element” on images, I can only do it on the menu or on the whole page "behind" the menu. (I guess in a way, images are behind the menu but the rendering show them instead of the menu. I've got really no experience with web dev so I don't know what I'm saying.)

![bugmastoimage6bis](https://user-images.githubusercontent.com/48503235/92746428-91c39200-f383-11ea-9b16-ad09a7e8e63d.png)
![bugmastoimage7bis](https://user-images.githubusercontent.com/48503235/92746739-cfc0b600-f383-11ea-8fad-bd5c53e6b3b2.png)
